src/modules/scrolling/auto-scroller.js

```diff
@@ -122,12 +134,26 @@ class AutoScroller {
         return true;
       return false;
     };
-    if (!this.options.stopScrollingOnHover.value) return;
-    this.isCursorOnText = isCursorOnTextFunc(ev);
+    if (this.isEnabledStopScrollingOnHover()) return;
+    this.cursorOnText = isCursorOnTextFunc(ev);
   }
 
   onMouseoutListener() {
-    this.isCursorOnText = false;
+    this.cursorOnText = false;
+  }
+
+  onWheelListener(ev) {
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Warning - 'ev' is defined but never used. (no-unused-vars)